samples/highcharts/demo/series-table/demo.details

```diff
@@ -0,0 +1,10 @@
+---
+name: Series table
+authors:
+  - Sophie Bremer
+js_wrap: b
+tags:
+  - Highcharts demo
+categories:
+  - Dynamic charts
```

Comment:
Should this sample be in www.highcharts.com/demo at this point? Also, it's not working... JS error on adding row.